nextflow.config

```diff
@@ -1,15 +1,16 @@
 params {
-    outdir = "data/results"
+    regulon_net = "dorothea" // can bei either "collectri" or "dorothea"
```

Comment:
this is not used in your version?

Reply:
It is used to create the output directory, but not for analyses/10_mofa/11_easier.R. 

- Within analyses/10_mofa/11_easier.R, the code must be modified to incorporate the regulon_net parameter as in the `revision` branch